### PR TITLE
[POC] Registering

### PR DESCRIPTION
**Pull request by davidsanfal**

### CODE REVIEW

**Review by franramirez688 — APPROVED** [review]

File: conanio/pages/private/conan-token-generator.tsx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        `${encodeURI(urls.api.public)}/${encodeURI(urls.signup)}?full_name=${encodeURIComponent(
+          fullName,
+        )}&email=${encodeURIComponent(email)}`,
+        {
+          method: "POST",
+        },
+      );
+      const data = await response.json();
+      console.log(data)
+      console.log(response)
```
Comment (franramirez688) [inline conanio/pages/private/conan-token-generator.tsx:36]:
```suggestion
```

File: conanio/pages/private/conan-token-generator.tsx [hunk 2]
```diff
@@ -0,0 +1,187 @@
+import React, { FormEvent, ChangeEvent, useState } from "react";
```
Comment (czoido) [inline conanio/pages/private/conan-token-generator.tsx:1]: I think we better change the name from `conan-token-generator` to `register` or something like that?

File: conanio/pages/private/conan-token-generator.tsx [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+  };
+
+  return (
+    <React.StrictMode>
+      <div className="flex-wrapper bg-conan-blue">
+        <ConanCenterHeader />
+        <br />
+        <Container className="conancontainer">
+          <h1 className="text-center">
+            Welcome to the conan token generator page
```
Comment (czoido) [inline conanio/pages/private/conan-token-generator.tsx:54]:
```suggestion
            Registration
```

Comment (czoido) [inline conanio/pages/private/conan-token-generator.tsx:54]:
```suggestion
            Registration
```

File: conanio/pages/private/conan-token-generator.tsx [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+          <br />
+          <br />
+        </Container>
+        <ConanFooter />
+      </div>
+    </React.StrictMode>
+  );
+};
+
+export default ConanTokenGenerator;
```
Comment (czoido) [inline conanio/pages/private/conan-token-generator.tsx:187]: can we change this for something like `ConanCatalogRegistration` or something like that?

**Review by czoido — APPROVED** [review]

**Pull request merged**